pkg/build/nodeimage/build.go

```diff
@@ -46,21 +47,43 @@ func Build(options ...Option) error {
 		ctx.logger.Warnf("unsupported architecture %q", ctx.arch)
 	}
 
-	// locate sources if no kubernetes source was specified
-	if ctx.kubeRoot == "" {
-		kubeRoot, err := kube.FindSource()
+	if ctx.buildType == "directory" {
```

Comment:
I think we can omit this if `release` accepts a parameter that allows to override the default url

`https://dl.k8s.io` can be overriden via `file://local`

Reply:
i removed buildType totally from the PR. just 2 things now ... either a semver or a path-to-a-tgz-file.